core/settings.py

```
@@ -0,0 +1,652 @@
+            "corsheaders.middleware.CorsMiddleware",
+            "django.middleware.common.CommonMiddleware",
+            "django.middleware.csrf.CsrfViewMiddleware",
+            "django.contrib.auth.middleware.AuthenticationMiddleware",
+            "django.contrib.messages.middleware.MessageMiddleware",
+            "django.middleware.clickjacking.XFrameOptionsMiddleware",
+            # "django.middleware.cache.FetchFromCacheMiddleware",
+            "django_prometheus.middleware.PrometheusAfterMiddleware",
+        ],
+        environ=False,
+    )
+    DEBUG = values.BooleanValue(True, environ=False)
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)